### PR TITLE
added promises

### PR DESCRIPTION
**Pull request by silchyn**
[DEMO](https://silchyn.github.io/js_promises_practice_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1 +1,64 @@
...
+});
+const rightClickPromise = new Promise(resolve => {
+  document.addEventListener('mousedown', e => {
+    e.preventDefault();
+
+    if (e.button === 2) {
+      resolve();
+    }
+  });
+});
```
Comment (BudnikOleksii) [inline src/scripts/main.js:55]: You can do it in one handler by sending 'e.button'  that clicked in the parameter, the same code

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Good! You can improve your solution, left a comment😉

**Review by BudnikOleksii — APPROVED** [review]
🔥

File: src/scripts/main.js [hunk 2]
```diff
@@ -1 +1,57 @@
...
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(reject, 3000, 'First promise was rejected');
+});
+const secondPromise = new Promise(resolve => {
+  document.addEventListener('mousedown', e => {
+    e.preventDefault();
```
Comment (ivan-kadykalo) [inline src/scripts/main.js:31]: Avoid of using one letter names

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]

**Review by Ihor-Karpyn — APPROVED** [review]

**Review by ivan-kadykalo — APPROVED** [review]

**Pull request closed without merging**